src/hooks/useHandGestures.js

```diff
@@ -0,0 +1,130 @@
+import React, {useState, useEffect} from "react";
+import * as handTrack from "handtrackjs";
+import {Spinner} from "reactstrap";
+
+export const useHandGestures = () => {
+
+    const [x, setX] = useState('');
+    const [y, setY] = useState('');
+    const [model, setModel] = useState(null);
+
+    const video = document.getElementById('video');
+    const canvas = document.getElementById('video-canvas');
+    const context = canvas && canvas.getContext("2d");
+
+    const [ctx, setCtx] = useState(null);
+
+    const modelParams = {
+        flipHorizontal: true, // flip e.g for video
+        imageScaleFactor: 0.7, // reduce input image size for gains in speed.
+        maxNumBoxes: 20, // maximum number of boxes to detect
+        iouThreshold: 0.5, // ioU threshold for non-max suppression
```

Comment:
What's ioU stand for?

Reply:
IOU calculation is actually used to measure the overlap between two proposals. Wiki :)